app/src/main/java/com/nervousfish/nervousfish/activities/RhythmCreateActivity.java

```diff
@@ -0,0 +1,186 @@
+package com.nervousfish.nervousfish.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.Button;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+import com.nervousfish.nervousfish.R;
+import com.nervousfish.nervousfish.data_objects.Contact;
+import com.nervousfish.nervousfish.data_objects.Profile;
+import com.nervousfish.nervousfish.data_objects.SimpleKey;
+import com.nervousfish.nervousfish.data_objects.tap.SingleTap;
+import com.nervousfish.nervousfish.modules.pairing.events.NewDataReceivedEvent;
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+
+import org.greenrobot.eventbus.Subscribe;
+import org.greenrobot.eventbus.ThreadMode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * The RhythmCreateActivity is an Activity where you can tap a sequence.
+ */
+//List is cast to an ArrayList, but that is needed to put in an intent
```

Comment:
I'm not sure why the List cant be put in the intent. I searched internet but everyone said that you should use the implementation of List instead of the super class.